CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
+### Bug fixes
+- Reverse order of arguments for `operator.contains` ([#9192](https://github.com/astral-sh/ruff/pull/9192))
+- Iterate over lambdas in deferred type annotations ([#9175](https://github.com/astral-sh/ruff/pull/9175))
+- Fix panic in D208 with multibyte indent ([#9147](https://github.com/astral-sh/ruff/pull/9147))
+- Add support for `NoReturn` in auto-return-typing ([#9206](https://github.com/astral-sh/ruff/pull/9206))
+- Allow removal of `typing` from `exempt-modules` ([#9214](https://github.com/astral-sh/ruff/pull/9214))
+- Avoid `mutable-class-default` violations for Pydantic subclasses ([#9187](https://github.com/astral-sh/ruff/pull/9187))
+- Fix dropped union expressions for piped non-types in `PYI055` autofix ([#9161](https://github.com/astral-sh/ruff/pull/9161))
+- Enable annotation quoting for multi-line expressions ([#9142](https://github.com/astral-sh/ruff/pull/9142))
+- Deduplicate edits when quoting annotations ([#9140](https://github.com/astral-sh/ruff/pull/9140))
+- Prevent invalid utf8 indexing in cell magic detection ([#9146](https://github.com/astral-sh/ruff/pull/9146))
+- Avoid nested quotations in auto-quoting fix ([#9168](https://github.com/astral-sh/ruff/pull/9168))
+- Add base-class inheritance detection to flake8-django rules ([#9151](https://github.com/astral-sh/ruff/pull/9151))
+- Avoid `asyncio-dangling-task` violations on shadowed bindings ([#9215](https://github.com/astral-sh/ruff/pull/9215))
+
+### Documentation
+- Fix blog post URL in changelog ([#9119](https://github.com/astral-sh/ruff/pull/9119))
+- Add error suppression hint for multi-line strings ([#9205](https://github.com/astral-sh/ruff/pull/9205))
+- Fix typo in SemanticModel.parent_expression docstring ([#9167](https://github.com/astral-sh/ruff/pull/9167))
+- Document link between import sorting and formatter ([#9117](https://github.com/astral-sh/ruff/pull/9117))
+
+### Other changes
+- Bump once_cell from 1.18.0 to 1.19.0 ([#9183](https://github.com/astral-sh/ruff/pull/9183))
+- Split string formatting to individual nodes ([#9058](https://github.com/astral-sh/ruff/pull/9058))
+- Add new `with` and `match` sequence test cases ([#9128](https://github.com/astral-sh/ruff/pull/9128))
+- [`flake8-bugbear`] Add fix for `zip-without-explicit-strict` (`B905`) ([#9176](https://github.com/astral-sh/ruff/pull/9176))
+- Bump dawidd6/action-download-artifact from 2 to 3 ([#9178](https://github.com/astral-sh/ruff/pull/9178))
+- Bump unicode_names2 from 1.2.0 to 1.2.1 ([#9184](https://github.com/astral-sh/ruff/pull/9184))
+- release: switch to Cargo's default ([#9031](https://github.com/astral-sh/ruff/pull/9031))
+- Extend `can_omit_optional_parentheses` documentation ([#9127](https://github.com/astral-sh/ruff/pull/9127))
+- Add fix to automatically remove `print` and `pprint` statements ([#9208](https://github.com/astral-sh/ruff/pull/9208))
+- Update ecosystem check headers to show unchanged project count ([#9157](https://github.com/astral-sh/ruff/pull/9157))
```

Comment:
Similarly we can drop internal changes related to CI, we should add the `ci` label to the excluded set in our rooster settings in the `pyproject.toml`